CHANGES

```diff
@@ -4,6 +4,7 @@
 - Scene2d.ui: Added new ParticleEffectActor to use particle effects on Stage
 - API addition: iOS: Added HdpiMode option to IOSApplicationConfiguration to allow users to set whether they want to work in logical or raw pixels (default logical).
 - Fix for #6377 Gdx.net.openURI not working with targetSdk 30
+- ObjLoader: Added ambient color and better texture handling
```

Comment:
it should be more precise, something like "ObjLoader now support ambientColor, ambientTexture, etc..."

Thing is default libGDX shader doesn't support most of these (only specular texture if i'm not wrong). It could be useful for custom shaders though.